gazebosim/gz-rendering
# Fix destroying wide angle camera

# 🦟 Bug fix

This fixes gz-sensors7's `INTEGRATION_wide_angle_camera_test` for me. 

To test:

1. build gz-sensors7 in your worksapce
2. Runt the test with `ogre` render engine from your colcon workspace:

    ```
    RENDER_ENGINE_VALUES=ogre ./build/gz-sensors7/bin/INTEGRATION_wide_angle_camera
    ```

    The test should now pass instead of segfault.

## Summary

## Checklist

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.
